corehq/motech/dhis2/tasks.py

```diff
@@ -195,6 +201,19 @@ def get_date_filter(report_config):
     return next((f for f in report_config.filters if f['type'] == 'date'), None)
 
 
+def get_previous_week(send_date: date) -> DateSpan:
+    """
+    Returns a DateSpan from last week Monday to last week Sunday
+
+    ISO 8601 has Monday as Day 1 and Sunday as Day 7
+    """
+    # monday.weekday() == 0
```

Comment:
Should this be removed? Looks like it is different to what you wrote in the docstring above

Reply:
ISO 8601 has Monday as 1 but Python has it as 0. I'll clarify the comment.